Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

This is added to fix the MKDEP value for Solaris Studio compiler.

--- build/apr_common.m4
+++ build/apr_common.m4
@@ -959,7 +959,7 @@
   int main() { return 0; }
 EOF
   MKDEP="true"
-  for i in "$CC -MM" "$CC -M" "$CPP -MM" "$CPP -M" "cpp -M"; do
+  for i in "$CC -MM" "$CC -M" "$CC -xM" "$CPP -MM" "$CPP -M" "$CPP -xM" "cpp -M"; do
     AC_MSG_CHECKING([if $i can create proper make dependencies])
     if $i conftest.c 2>/dev/null | grep 'conftest.o: conftest.c' >/dev/null; then
       MKDEP=$i
